sdk/servicebus/service-bus/samples/javascript/advanced/deferral.js

```diff
@@ -1,17 +1,17 @@
 /*
-Copyright (c) Microsoft Corporation. All rights reserved.
-Licensed under the MIT Licence.
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
 
-This sample demonstrates how the defer() function can be used to defer a message for later processing.
+  This sample demonstrates how the defer() function can be used to defer a message for later processing.
 
-In this sample, we have an application that gets cooking instructions out of order. It uses
-message deferral to defer the instruction that is out of order, and then processes it in order.
+  In this sample, we have an application that gets cooking instructions out of order. It uses
+  message deferral to defer the instruction that is out of order, and then processes it in order.
 
-See https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-deferral to learn about
-message deferral.
+  See https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-deferral to learn about
+  message deferral.
```

Comment:
We should have a note here on all samples pointing people to the older samples folder if they are using v1. This is to save anyone landing here but are using v1

Reply:
Added (it's pretty much the same note as what we've used in the readme)

I also updated the 3 other readmes for the samples to include a shortened version of the same.